dnf-behave-tests/dnf/downgrade.feature

```diff
@@ -55,6 +55,14 @@ Scenario: Downgrade RPM that requires downgrade of dependency
         | glibc-all-langpacks-2.28-9.fc29.x86_64 | Dependency | dnf-ci-fedora |
         | glibc-common-2.28-9.fc29.x86_64        | Dependency | dnf-ci-fedora |
         | setup-2.12.1-1.fc29.noarch             | Dependency | dnf-ci-fedora |
+    And transaction items for transaction "last" are
```

Comment:
Because the check is only for DNF5 it would be nice to add that information even to the step syntax. And because the check uses history output it could be beneficial to somehow mention it.
What about `history transaction items for DNF5 transaction "last" are`? But there are multiple ways how it can be formulated.

Reply:
I've prepended "dnf5 " to the name of the step, we already have "dnf5 stdout/stderr is"...